### PR TITLE
Kelly task solution for week 1

### PR DESCRIPTION
**Pull request by Kelyxa**

### CODE REVIEW

File: src/week1/kelly/DivWithoutOperator_kelly.java
Comment (Suna898789) [inline src/week1/kelly/DivWithoutOperator_kelly.java:1]: good job

File: src/week1/kelly/Finra_kelly.java
Comment (Suna898789) [inline src/week1/kelly/Finra_kelly.java:1]: nice approach

File: src/week1/kelly/OddEven_kelly.java
Comment (Suna898789) [inline src/week1/kelly/OddEven_kelly.java:1]: nice job

**Review by Suna898789 — APPROVED** [review]
Good job

File: src/week1/kelly/Finra_kelly.java
Comment (Kelyxa) [reply src/week1/kelly/Finra_kelly.java:1]: thank you!

**Pull request merged**